skymap_scanner/server/start_scan.py

```diff
@@ -725,13 +732,15 @@ def _nside_and_pixelextension(val: str) -> Tuple[int, int]:
         name=f"to-clients-{scan_id}",
         auth_token=cfg.ENV.SKYSCAN_BROKER_AUTH,
         timeout=cfg.ENV.SKYSCAN_MQ_TIMEOUT_TO_CLIENTS,
+        ack_timeout=cfg.SERVER_ACK_TIMEOUT,
     )
     from_clients_queue = mq.Queue(
         cfg.ENV.SKYSCAN_BROKER_CLIENT,
         address=cfg.ENV.SKYSCAN_BROKER_ADDRESS,
         name=f"from-clients-{scan_id}",
         auth_token=cfg.ENV.SKYSCAN_BROKER_AUTH,
         timeout=cfg.ENV.SKYSCAN_MQ_TIMEOUT_FROM_CLIENTS,
+        ack_timeout=cfg.SERVER_ACK_TIMEOUT,
```

Comment:
We don't want this. The server should have short heartbeats to avoid TCP shenanigans. Call `connection.process_data_events()` occasionally (perhaps abstract away to mqclient). This issue we're trying to address is when the server has transitioned to the collection phase while the sending-connection is sitting idle

Reply:
Being fixed in https://github.com/Observation-Management-Service/ewms-pilot/pull/44